cli/cli.go

```diff
@@ -38,11 +36,24 @@ func Run() {
 			Name:  "experimental",
 			Usage: "enable experimental features",
 		},
+		cli.StringFlag{
+			Name:  "log-file",
+			Usage: "Logging the output to a file instead of stdout",
+		},
 	}
-
 	// logs
 	app.Before = func(c *cli.Context) error {
 		log.SetOutput(os.Stderr)
+		if c.IsSet("log-file") {
+			logDir := c.String("log-file")
+			file, err := os.OpenFile(logDir, os.O_RDWR, 0777)
```

Comment:
yes I think it would be nice to close it in `app.After`
Manage never exit anyway, but it would be nice to do that for other subcommands.

Also, (in another PR) we might catch signals and do a proper cleanup